clients/interfaces/command.go

```diff
@@ -23,8 +23,8 @@ type CommandClient interface {
 	// DeviceCoreCommandsByDeviceName returns all commands associated with the specified device name.
 	DeviceCoreCommandsByDeviceName(ctx context.Context, deviceName string) (responses.DeviceCoreCommandResponse, errors.EdgeX)
 	// IssueGetCommandByName issues the specified read command referenced by the command name to the device/sensor that is also referenced by name.
-	// dsPushEvent: If set to yes, a successful GET will result in an event being pushed to the EdgeX system. Default value is no.
-	// dsReturnEvent: If set to no, there will be no Event returned in the http response. Default value is yes.
+	// dsPushEvent: If set to true, a successful GET will result in an event being pushed to the EdgeX system. Default value is false.
+	// dsReturnEvent: If set to false, there will be no Event returned in the http response. Default value is true.
 	IssueGetCommandByName(ctx context.Context, deviceName string, commandName string, dsPushEvent string, dsReturnEvent string) (*responses.EventResponse, errors.EdgeX)
```

Comment:
Can we make this interface use `bool` instead of `string` for these options? 
```suggestion
	IssueGetCommandByName(ctx context.Context, deviceName string, commandName string, dsPushEvent bool, dsReturnEvent bool) (*responses.EventResponse, errors.EdgeX)
```